docs/source/data/resolvers.mdx

```diff
@@ -388,49 +374,75 @@ Resolver functions are passed four arguments: `parent`, `args`, `context`, and `
 
 > You can use any name for each argument in your code, but the Apollo docs use these names as a convention. Instead of `parent`, it's also common to use the parent type's name or `source`.
 
-| Argument  | Description  |
-|---|---|
-| `parent` | <p>The return value of the resolver for this field's parent (i.e., the previous resolver in the [resolver chain](#resolver-chains)).</p><p>For resolvers of top-level fields with no parent (such as fields of `Query`), this value is obtained from the `rootValue` function passed to [Apollo Server's constructor](../api/apollo-server/#constructor).</p>  |
-| `args` |  <p>An object that contains all GraphQL arguments provided for this field.</p><p> For example, when executing `query{ user(id: "4") }`, the `args` object passed to the `user` resolver is `{ "id": "4" }`.</p> |
-| `context` | <p>An object shared across all resolvers that are executing for a particular operation. Use this to share per-operation state, including authentication information, dataloader instances, and anything else to track across resolvers. </p><p>See [The `context` argument](#the-context-argument) for more information.</p> |
-| `info` | <p>Contains information about the operation's execution state, including the field name, the path to the field from the root, and more. </p><p>Its core fields are listed in the [GraphQL.js source code](https://github.com/graphql/graphql-js/blob/main/src/type/definition.ts#L977-L986). Apollo Server extends it with a [`cacheControl` field](../performance/caching/#in-your-resolvers-dynamic).</p> |
+| Argument  | Description                                                                                                                                                                                                                                                                                                                                                                                                                |
+| --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
prettier? more like uglier

What mechanism are you using to run prettier (or a different formatter) btw? We have `*.md` and `*.mdx` in our prettierignore file — my VSCode is set up to not format-on-save. Though I like prettier and would be happier to add

```
!docs/source/**/*.md
!docs/source/**/*.mdx
```

to the bottom of that file (then run `npm run prettier-fix` once).

It also looks like if we add `"proseWrap": "never" to `.prettierrc.json5` then it [formats wide tables better](https://github.com/prettier/prettier/pull/5701).

Reply:
😂  Oh geez I didn't realize it'd get this bad! My VSCode prettier settings are now set to not format Markdown files, thank you for catching this!